src/bosh.js

```diff
@@ -886,7 +888,9 @@ Strophe.Bosh.prototype = {
     _send: function () {
         clearTimeout(this._conn._idleTimeout);
         this._throttledRequestHandler();
-        this._conn._idleTimeout = setTimeout(this._conn._onIdle.bind(this._conn), 100);
+        this._conn._idleTimeout = setTimeout(function() {
+            this._onIdle();
```

Comment:
We bind the anonymous function to `this._conn`, Ie. `function() {...}.bind(this._conn)`, hence `this` refers to `this._conn`